### PR TITLE
Punctuations edits from issue #799

### PR DESCRIPTION
**Pull request by MartinKoob**
Adding missing periods to sentences in **Description** and **Value Space** fields as well as converting one list of values from a blockquote to a list for consistency.

### CODE REVIEW

**Comment by MrBillMcDonald** [comment]
Per sub-group meeting ready to submit to monthly meeting.

File: cmi5_spec.md [hunk 1]
```diff
@@ -1541,14 +1540,7 @@ The data in this section are used by the LMS to locate the AU and provide launch
...
       <p>If all member AUs in a block are satisfied, then the block is considered satisfied for prerequisites and sequencing.<br>If all member AUs and Blocks are satisfied, then the course is considered satisfied for prerequisites or credit in relation to other courses or curricula.</p>
-      <p><strong>Value space:</strong></p>
-      <blockquote>
-        <p>"Passed"<br>
-          "Completed"<br>
-          "CompletedAndPassed"<br>
-          "CompletedOrPassed"<br>
-          "NotApplicable"</p>
-      </blockquote>
+      <p><strong>Value space:</strong>"Passed", "Completed", "CompletedAndPassed", CompletedOrPassed, "NotApplicable"</p>
```
Comment (brianjmiller) [inline cmi5_spec.md]:
```suggestion
      <p><strong>Value space:</strong>"Passed", "Completed", "CompletedAndPassed", "CompletedOrPassed", "NotApplicable"</p>
```

**Review by brianjmiller — CHANGES_REQUESTED** [review]

**Review by brianjmiller — APPROVED** [review]

File: cmi5_spec.md [hunk 2]
```diff
@@ -1541,14 +1540,7 @@ The data in this section are used by the LMS to locate the AU and provide launch
...
       <p>If all member AUs in a block are satisfied, then the block is considered satisfied for prerequisites and sequencing.<br>If all member AUs and Blocks are satisfied, then the course is considered satisfied for prerequisites or credit in relation to other courses or curricula.</p>
-      <p><strong>Value space:</strong></p>
-      <blockquote>
-        <p>"Passed"<br>
-          "Completed"<br>
-          "CompletedAndPassed"<br>
-          "CompletedOrPassed"<br>
-          "NotApplicable"</p>
-      </blockquote>
+      <p><strong>Value space:</strong>"Passed", "Completed", "CompletedAndPassed", "CompletedOrPassed", "NotApplicable"</p>
```
Comment (brianjmiller) [inline cmi5_spec.md]:
```suggestion
      <p><strong>Value space: </strong>"Passed", "Completed", "CompletedAndPassed", "CompletedOrPassed", "NotApplicable"</p>
```

**Comment by MrBillMcDonald** [comment]
Approved per Aug-4-2023 Meeting

**Pull request merged**